pkg/controller/volume/persistentvolume/delete_test.go

```diff
@@ -148,9 +148,9 @@ func TestDeleteSync(t *testing.T) {
 		},
 		{
 			// PV requires external deleter
-			name:            "8-10 - external deleter",
-			initialVolumes:  []*v1.PersistentVolume{newExternalProvisionedVolume("volume8-10", "1Gi", "uid10-1", "claim10-1", v1.VolumeBound, v1.PersistentVolumeReclaimDelete, classEmpty, gceDriver, nil, volume.AnnBoundByController)},
-			expectedVolumes: []*v1.PersistentVolume{newExternalProvisionedVolume("volume8-10", "1Gi", "uid10-1", "claim10-1", v1.VolumeReleased, v1.PersistentVolumeReclaimDelete, classEmpty, gceDriver, nil, volume.AnnBoundByController)},
+			name:            "8-10-1 - external deleter when volume is provisioned by controller",
```

Comment:
Could you clarify what "volume is provisioned by controller" means?

Reply:
It refers to dynamic volume provisioning. User create a pv with a storageclass. kube-controller-mananger will trigger a pv creation and the created pvc will be annotated by `pv.kubernetes.io/provisioned-by`

https://github.com/kubernetes/kubernetes/blob/cc8a7e812ec5b7256de470021107cd849ceda691/pkg/controller/volume/persistentvolume/delete_test.go#L158-L163

Reply:
Please change it to dynamic volume provisioning itself.